.github/workflows/rust.yml

```diff
@@ -87,8 +86,4 @@ jobs:
       - uses: cachix/install-nix-action@v16
         with:
           nix_path: nixpkgs=channel:nixos-22.05
-      - uses: cachix/cachix-action@v10
-        with:
-          name: mini
-          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
-      - run: nix-build default.nix
+      - run: nix build
```

Comment:
Isn't the whole point of this making caching better?

Reply:
cachix was pullying older derivations so masked it for a while, we will unmask it before this gets merged